learning_wavelets/evaluate/exact_recon_unet_evaluate.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import numpy as np
+import tensorflow as tf
+from tqdm import tqdm
+
+from runstats import Statistics
+from skimage.measure import compare_psnr, compare_ssim
+from learning_wavelets.config import LOGS_DIR, CHECKPOINTS_DIR
+from learning_wavelets.data.datasets import im_dataset_bsd68, im_dataset_bsd500
+
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+from learning_wavelets.evaluate.utils import METRIC_FUNCS, Metrics
+
+def evaluate_unet(
+        run_id = 'ExactReconUnet_4_bsd500_0_55_2000_1620051556-20',
+        n_epochs = 500,
+        n_output_channels = 1,
+        kernel_size = 3,
+        layers_n_channels = [64, 128, 256, 512, 1024],
+        layers_n_non_lins = 2,
+        non_linearity = 'relu',
```

Comment:
Don't forget about the [pep8 guidelines for spacing in kwargs](https://www.python.org/dev/peps/pep-0008/#other-recommendations)